docs/mobile-apps/automated-testing/about-appium.md

```diff
@@ -0,0 +1,437 @@
+
+This example below illustrates how you could use an explicit wait to anticipate web browser response after submitting the login form.
+
+```java
+import org.openqa.selenium.support.ui.ExpectedConditions; 
+import org.openqa.selenium.support.ui.WebDriverWait; 
+WebDriverWait wait = new WebDriverWait(driver, 10); WebElement messageElement = wait.until( ExpectedConditions.presenceOfElementLocated(By.id("loginResponse")) );
+```
+
+>**NOTE**: Do not mix implicit and explicit waits. Doing so can cause unpredictable wait times. For example setting an implicit wait of 10s and an explicit wait of 15 seconds, could cause a timeout to occur after 20 seconds.
+
+<br/>
+
+### 6. Run Tests and Record Your Results
+
+This is where you log whether your test passed, failed, or returned an error.
+
+Running tests and recording test results is the ultimate purpose of your test script: you run tests in an automated test script in order to evaluate function and performance in the AUT, without requiring human interaction.
+
+#### **Test Frameworks**
+
+To run test and to record test results, you use methods of a test framework for your programming language. There are many available test frameworks, including the frameworks in the so-called XUnit family, which includes:
+
+* JUnit for Java
+* NUnit for C#
+* unittest or pyunit for Python
+* RSpec for Ruby
+
+For some programming languages, test frameworks other than those in the XUnit family are common - for example, the RSpec framework for Ruby. Our [sample test framework repos on GitHub](https://github.com/saucelabs-training) contain over 60 examples of test frameworks set up to work with Sauce Labs. 
+
+#### **Test Assertions**
+
+Most test frameworks implement the basic concept of an ***assertion***, a method representing whether or not a logical condition holds after interaction with an AUT.
+
+Test frameworks generally declare methods whose names begin with the term `assert` and end with a term for a logical condition (e.g., `assertEquals` in JUnit). Generally, when the logical condition represented by an assert method does not hold, an exception for the condition is thrown.
+
+There are various approaches to using exceptions in most test frameworks. The [SeleniumHQ documentation](https://www.selenium.dev/documentation/en/) has more detailed information on using assertions and verifications in your tests. 
+
+#### **Recording Test Results**
+
+Recording of test results can be done in various ways, supported by the test framework or by a logging framework for the programming language, or by both together. Selenium also supports taking screenshots of web browser windows as a helpful additional type of recording.
```

Comment:
Why are we talking about Selenium here?

Reply:
I didn't make this doc- simply ported over from old appium.md that is currently published here (https://docs.saucelabs.com/mobile-apps/automated-testing/appium)-- feel free to add any copy-editing commits you would like